### ISSUE TITLE
Does not work with Rails 5.2.0.rc1

### ISSUE THREAD

**Issue by cjlarose** [mlitwiniuk/localized_country_select#27]
[This commit][0] breaks the use of the method `ActionView::Helpers::Tags::base#value`.

[0]: https://github.com/rails/rails/commit/530b5fea8f3e04c45a20ff3dcc8d473a6d25c1b4

I think calling `#value` with no arguments fixes it, but `object` needs to be passed for all previous versions of Rails

```diff
diff --git a/lib/localized_country_select.rb b/lib/localized_country_select.rb
index b2351ac..1ee748b 100644
--- a/lib/localized_country_select.rb
+++ b/lib/localized_country_select.rb
@@ -93,19 +93,18 @@ module ActionView
       end
       alias_method :country_options_for_select, :localized_country_options_for_select
 
     end
 
     module ToCountrySelectTag
       def to_localized_country_select_tag(priority_countries, options, html_options)
         html_options = html_options.stringify_keys
         add_default_name_and_id(html_options)
-        value = value(object)
         content_tag("select",
           add_options(
             localized_country_options_for_select(value, priority_countries, options).html_safe,
             options, value
           ), html_options
         )
       end
     end
```

**Comment by jschiefner** [comment]
@cjlarose calling value without arguments is already necessary in rails >= 5.1, in Rails 5.0.7 it still works with the argument.

**Comment by LiorPSHS** [comment]
Any chance of this gem being updated for Rails 5.2?

**Comment by cstyles** [comment]
I believe this has been resolved as of d8cea6861d9cfccf0bece78335fd789ec5e28a21
